pcap/bpf_test.go

```diff
@@ -0,0 +1,65 @@
+package pcap
```

Comment:
please add copyright notice.  See the one on https://github.com/google/gopacket/blob/master/layers/tls_appdata.go for an easily copy-paste-able version.  Feel free to add your name to the AUTHORS file at the top-level as well if you'd like.